flavors-modified-vs-master.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 SUSE LLC
+
+import os
+import re
+import subprocess
+import sys
+
+
+def _run_cmd(cmd_array):
+    return subprocess.check_output(cmd_array).decode("utf-8")
+
+
+# Stage the flavor and return the staging dir
+def _stage_flavor(flavor):
+    stage_output = _run_cmd(['make', 'FLAVORS_TO_BUILD='+flavor, 'stage'])
+    staging_dir_pattern = re.compile(r'^\s*STAGING_DIR\s*:\s+(.*)$', re.MULTILINE)
+    match = staging_dir_pattern.search(stage_output)
+    if not match or not match[1] or match[1] == '':
+        print("Could not find staging dir for:\n{}".format(stage_output))
+        sys.exit(1)
+    return match[1]
+
+
+def _filediff_intersects_sources(filediff, sources):
+    for f in filediff.splitlines():
+        file_pattern = re.compile(r'^.*\s+<-\s+'+f+'$', re.MULTILINE)
+        match = file_pattern.search(sources)
+        if match:
+            return True
+    return False
+
+
+# Get list of files different from master
+filediff = _run_cmd(['git', 'diff', '--name-only', 'master'])
```

Comment:
I think using `origin/master` will be more accurate than the local master branch.